darkmoon/api/v1/metadata/schema.py

```diff
@@ -131,3 +131,20 @@ class MetadataEntity(BaseModel):
         description="contains all the header information",
         example="",
     )
+
+
+class Response(BaseModel):
+    """Sets model for other reponse types to inherit from."""
+
+    message: str = Field(
+        description="Base response model",
+        example="",
+    )
+
+
+class UploadResponse(Response):
+    """Sets upload response requirements."""
+
+    object: Metadata = Field(
+        description="The object inserted or updated on the database",
+    )
```

Comment:
Change the term from object to data since it's more standard. You are not always getting 1 object, sometimes it's a list.